src/main/java/com/acoustic/salarycalculator/jobscategories/JobsCategory.java

```diff
@@ -0,0 +1,71 @@
+package com.acoustic.salarycalculator.jobscategories;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+
+@AllArgsConstructor
+@Getter
+public enum JobsCategory {
+
+    IT(1, "It", new ArrayList<>(List.of(
```

Comment:
why to use new ArrayList<>(List.of(...)
you could just use List.of(...)

Reply:
fixed